src/containers/community/CommunitySettings.js

```diff
@@ -20,21 +20,28 @@ import config from '../../config'
 const { host } = config
 const slackClientId = config.slack.clientId
 import { avatarUploadSettings, bannerUploadSettings } from '../../models/community'
+import { hasFeature } from '../../models/currentUser'
 import A from '../../components/A'
 import { uploadImage } from '../../actions/uploadImage'
 import PersonChooser from '../../components/PersonChooser'
 import { communityJoinUrl } from '../../routes'
 import { makeUrl } from '../../util/navigation'
+import InvitationList from './InvitationList'
+import JoinRequestList from './JoinRequestList'
 
 @prefetch(({dispatch, params: {id}}) =>
   Promise.all([
     dispatch(fetchCommunitySettings(id)),
-    dispatch(fetchCommunityModerators(id))
+    dispatch(fetchCommunityModerators(id)),
+    dispatch(fetchInvitations(id)),
+    dispatch(fetchJoinRequests(id))
```

Comment:
hmmm -- firing off four separate requests per page load here seems excessive. better to run each fetch only when its corresponding section is opened for the first time